Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -189,6 +228,37 @@ public bool TryGetNearGraspPoint(out Vector3 result)
             }
         }
 
+
+        private static readonly ProfilerMarker TryGetNearGraspAxisPerfMarker = new ProfilerMarker("[MRTK] ConePointer.TryGetNearGraspAxis");
+
+        /// <summary>
+        /// Gets the axis that the grasp happens
+        /// For the SpherePointer it's the axis from the palm to the grasp point
```

Comment:
It feels like it is going from palm joint to index tip, not grasp point. Is this correct?

Reply:
You're right I'll update the comment